protocols/server/cloud-events/README.md

```diff
@@ -0,0 +1,8 @@
+# Web PubSub CloudEvents HTTP extensions
+
+This project describes the Web PubSub HTTP protocols to communicate client events with upstream servers. **Currently only MQTT client events are described in the protocol.**
+
+* Typespec version: [`main.tsp`](./main.tsp)
```

Comment:
is the file name customizable? file naming: cloud-events-http-extensions.tsp?

Reply:
This is a typical typespec project: https://github.com/Azure/azure-rest-api-specs/tree/main/specification/ai/OpenAI.Assistants/files 

The difference with this one is that we combine models and operations into one file.